.github/workflows/integration-test-config-older-k8s/action.yaml

```diff
@@ -0,0 +1,37 @@
+name: Cluster Setup
```

Comment:
Didn't we decide that using the different older/newer config didn't help much? Or does this still help with k8s 1.17/1.18?

Reply:
I thought we decided to keep it in 1.9 for 1.18

Reply:
also, origin pr was merged in 1.10 branch too.